Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -1607,6 +1808,12 @@ void ACesium3DTileset::Tick(float DeltaTime) {
 
   updateTilesetOptionsFromProperties();
 
+  // TODO: Tileset traversal and occlusion history retrieval may use different
+  // cameras, look for ways to consolidate them.
```

Comment:
I think the aggregation we now do over all the view families from the ISceneViewExtension should give us all the relevant views. As discussed before, this may be more exhaustive than the list of views used for traversal, in a future PR we should use the view extension to determine views for the traversal as well.